script/utils/update-redirects-data.ts

```diff
@@ -42,6 +42,10 @@ const updateRedirectsData = async(currentBook: BookWithOSWebData, newBook: BookW
     ) {
       redirects.push(formatSection(section));
       countNewRedirections++;
+    } else if (!slug) {
+      // tslint:disable-next-line: no-console
+      console.error(`page with ID ${section.id} not found in new book`);
+      process.exit(1);
```

Comment:
process.exit will kill the whole script, you should throw an error like above to allow the parent script to aggregate errors from multiple books or whatever it does